pkg/kubelet/apis/podresources/grpc/ratelimit.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package grpc
+
+import (
+	"context"
+
+	gotimerate "golang.org/x/time/rate"
+	"k8s.io/klog/v2"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	// DefaultQPS is determined empirically reviewing known consumers of the API.
+	// It's at least unlikely that there is a legitimate need to query podresources
+	// more than 100 times per second, the other subsystems are not guaranteed to react
+	// so fast in the first place.
+	DefaultQPS = 100
+	// DefaultBurst is determined empirically reviewing lnown consumers of the API.
+	// See the documentation of DefaultQPS, same caveats apply.
+	DefaultBurst = 10
```

Comment:
```suggestion
	// DefaultQPS is determined by empirically reviewing known consumers of the API.
	// It's at least unlikely that there is a legitimate need to query podresources
	// more than 100 times per second, the other subsystems are not guaranteed to react
	// so fast in the first place.
	DefaultQPS = 100
	// DefaultBurst is determined by empirically reviewing known consumers of the API.
	// See the documentation of DefaultQPS, same caveats apply.
	DefaultBurst = 10
```

Also the semantics of burst is different than in https://github.com/kubernetes/kubernetes/blob/fcfe5dfc2177320833caa451b401f2d1dc5668e7/pkg/kubelet/apis/config/v1beta1/defaults.go#L202-L207. Perhaps this can be renamed to bucketTokens or burstTokens?

Reply:
+1 on Sergey above.

Reply:
Thanks Sergey and Dawn, changing per your suggestions. I'll review in more detail the semantics of burst. Disclosure: at first I understood it works like kube expects but indeed it seems there are differences I need to understand better. I still think this is good enough for the limited scope of the podresources requirements, but to generalize this solutions these differences needs to be understood and addressed.